packages/cards-editor-middleware/templates/Init.js

```diff
@@ -0,0 +1,84 @@
+function parseUI5Version(version) {
+    const versionParts = version.replace(/snapshot-untested|snapshot-|snapshot/, '').split('.');
+    const major = parseInt(versionParts[0], 10);
+    const minor = parseInt(versionParts[1], 10);
+
+    return { major, minor };
+}
+
+function isLowerThanMinimalUi5Version(version, minVersion) {
+    if (version && minVersion) {
+        const minVersionParsed = parseUI5Version(minVersion);
+        const ui5VersionParsed = parseUI5Version(version);
+        if (!isNaN(ui5VersionParsed.major) && !isNaN(ui5VersionParsed.minor)) {
+            if (ui5VersionParsed.major < minVersionParsed.major) {
+                return true;
+            }
+            if (ui5VersionParsed.major === minVersionParsed.major && ui5VersionParsed.minor < minVersionParsed.minor) {
+                return true;
+            }
+        }
+    }
+    return false;
+}
+
+function addCardGenerationUserAction(oComponentInstance) {
+    sap.ui.require([
+        "sap/cards/ap/generator/CardGenerator"
+    ], (CardGenerator) => {
+        sap.ui.require(["sap/ushell/Container"], async function (Container) {
```

Comment:
Can't you just require both libs instead of nesting the require?

```js
sap.ui.require([
        "sap/cards/ap/generator/CardGenerator",
        "sap/ushell/Container"
    ], (CardGenerator, Container) => {
...
```